pegasus/sites.v3/code.org/public/dance-landing.haml

```diff
@@ -0,0 +1,33 @@
+%h1= I18n.t(:hoc2018_dance_party_main_heading)
+
+.dance-subheading
+  %h4= I18n.t(:hoc2018_dance_party_subheading)
+
+%link{href: "/css/dance-landing.css", rel: "stylesheet"}
+
+%div{style: "clear:both"}
+
+.dance-container
+  .img-container
+    %img.dance-img{src:"/shared/images/fit-150/hadi_partovi.jpg"}
+  %h1.dance-heading= I18n.t(:hoc2018_dance_create_your_own_dance_title)
+
+  .col-25.dance-tutorial.dance-blocks-tutorial
+    .tutorial-box
+      %h2.dance-h2= I18n.t(:hoc2018_dance_party_box_title)
+      .dance-tutorial-info
+        %p.dance-tutorial-description= I18n.t(:hoc2018_dance_party_box_description)
+        %p.dance-tutorial-specs= I18n.t(:hoc2018_dance_party_specs)
+      %a{:href => '/api/hour/begin/starwarsblocks', :target=>'_self'}
```

Comment:
You should be able to hit this api with dance party so you can update here and other places in the file